virocon/contours.py

```diff
@@ -228,8 +334,41 @@ def _compute(self, ):
 
 
 class HighestDensityContour(Contour):
+    """
+        Contour based on the highest density method.
```

Comment:
different indentation, compared to the contours above

also attributes, see IFORM comment